src/components/ArticleDetailPage.jsx

```diff
@@ -20,16 +23,28 @@ export default function ArticleDetailPage() {
 				setLoading(false);
 			});
 
+		axios
```

Comment:
You might want to separate this into another component and have it's own loading state so people can see that you are loading the comments